client/src/components/HabitCard/HabitCard.js

```diff
@@ -24,4 +34,4 @@ export default function MediaCard() {
       </CardActions>
```

Comment:
Should we add each answer option to the cards as buttons too?

Let's map inside the `habit.answerOptions` and create a button as many as an option we have and give buttons the option name.

*For example:*

Let's say this is the data:

```json
{
    "_id": "6397f7beecd53351ef528051",
    "category": "Recycle",
    "description": "Reduce the amount of waste by using reusable bags!",
    "question": "What kind of bag did you use today?",
    "answerOptions": [
        "Reusable",
        "Plastic",
        "Both",
        "None"
    ],
    "__v": 0
}
```

Instead of "Share" and "Learn More," it should render: `Reusable`,`Plastic`, `Both`, `None`.